.travis.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   include:
     - env:
       - PYTHON=2.7
-      - NUMPY=1.12.1
+      - NUMPY=1.14.1
```

Comment:
For consistency, should maybe bump the 1.14.0 in the py36 env to 1.14.1 as well.

Reply:
Done